package.json

```diff
@@ -390,6 +390,7 @@
     "typeface-source-sans-pro": "^1.1.5",
     "typescript": "^3.9.5",
     "typescript-snapshots-plugin": "^1.7.0",
+    "url-loader": "4.1.1",
```

Comment:
This is needed by webpack, and had apparently been available because of docz, which has been removed.